File: app/src/main/java/foocafe/org/foocafe/entities/Badge.java
package foocafe.org.foocafe.entities;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Badge implements Serializable {

    @SerializedName("name")
    public String name;

    @SerializedName("image")
    public String image;

    @SerializedName("description")
    public String description;

    @SerializedName("criteria")
    public String criteria;
}

File: app/src/main/java/foocafe/org/foocafe/entities/Badge.kt
package foocafe.org.foocafe.entities

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Badge : Serializable {

    @SerializedName("name")
    var name: String? = null

    @SerializedName("image")
    var image: String? = null

    @SerializedName("description")
    var description: String? = null

    @SerializedName("criteria")
    var criteria: String? = null
}